### DISCUSSION TITLE
Disqus color on dark mode

### DISCUSSION THREAD

**Discussion by ms2063** [jpanther/congo#814] [Q&A]
Thanks for the wonderful theme that you made.

While I was applying the theme to my website I noticed that when I see disqus on dark mode the font color is a bit dark so is there anyway I can change the color to more brighter color?

<img width="759" alt="Screenshot 2024-02-23 at 17 49 47" src="https://github.com/jpanther/congo/assets/157222473/68d1cd13-c782-41d0-aebd-96ccb14d89fc">

**Comment by wolfspyre** [comment]
is your site online someplace we can inspect what you're seeing? 
(I've not personally set up disqus anywhere so not sure what attr's feeding it's color)
